packages/devtools_app/test/inspector/inspector_screen_test.dart

```diff
@@ -299,6 +295,46 @@ void main() {
     });
   });
 
+  group(
+    'FlutterInspectorSettingsDialog',
+    () {
+      const startingHoverEvalModeValue = false;
+
+      setUp(() {
+        inspectorPreferences.toggleHoverEvalMode(startingHoverEvalModeValue);
+      });
+
+      testWidgetsWithWindowSize(
+          'can update hover inspection setting', windowSize,
+          (WidgetTester tester) async {
+        await tester.pumpWidget(buildInspectorScreen());
+        await tester.pump();
```

Comment:
nit: why this extra pump?

Reply:
good call this may have been added while I was having an issue with testing